toolbox/linux/how.rst

```diff
@@ -0,0 +1,107 @@
+OK I'm Ready... How Do I Try/Install It?
+========================================
+
+So you're ready to install a Linux? Great! If it's your first time, hopefully
+you're starting with Ubuntu. Also if it's your first time, you may want to grab
+an experienced Linux buddy (ask in Slack or mailing list) to help you. Don't do
+this if you know you'll desperately need your computer the next day!
+
+*If you're not installing Ubuntu, these steps may not apply perfectly to you.*
+The information on virtual machines, live CDs, and live USBs will probably be
+helpful though.
+
+Step 1: Download
```

Comment:
So the only way people would really get to this page is by going to the index page, where they would see the sections in order.  In fact, the pages even have "next" and "previous" lengths...  I feel like this shouldn't be necessary.  Maybe pull this branch, clone it, build the HTML, and see how you feel about it?